lib/formatters/codeframe.js

```diff
@@ -4,9 +4,10 @@
  */
 "use strict";
 
-const chalk = require("chalk");
-const codeFrame = require("babel-code-frame");
+const os = require("os");
 const path = require("path");
+const chalk = require("chalk");
+const codeFrame = require("@babel/code-frame");
```

Comment:
What are the advantages of `@babel/code-frame` over `babel-code-frame`? Is there any difference?

Reply:
LTS